[2024-06-27T04:39:08.217+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-27T04:39:08.256+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final.insert_new_record_into_audit_table scheduled__2024-06-27T04:37:00+00:00 [queued]>
[2024-06-27T04:39:08.273+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final.insert_new_record_into_audit_table scheduled__2024-06-27T04:37:00+00:00 [queued]>
[2024-06-27T04:39:08.273+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-27T04:39:08.293+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_new_record_into_audit_table> on 2024-06-27 04:37:00+00:00
[2024-06-27T04:39:08.299+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=189) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-27T04:39:08.301+0000] {standard_task_runner.py:63} INFO - Started process 191 to run task
[2024-06-27T04:39:08.302+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_final', 'insert_new_record_into_audit_table', 'scheduled__2024-06-27T04:37:00+00:00', '--job-id', '4525', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting.py', '--cfg-path', '/tmp/tmpkr6ec_ad']
[2024-06-27T04:39:08.303+0000] {standard_task_runner.py:91} INFO - Job 4525: Subtask insert_new_record_into_audit_table
[2024-06-27T04:39:08.358+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_final.insert_new_record_into_audit_table scheduled__2024-06-27T04:37:00+00:00 [running]> on host eeacb89bcfc1
[2024-06-27T04:39:08.488+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_final' AIRFLOW_CTX_TASK_ID='insert_new_record_into_audit_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-27T04:37:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-27T04:37:00+00:00'
[2024-06-27T04:39:08.489+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-27T04:39:08.508+0000] {logging_mixin.py:188} INFO - insert_into_audit_table_for_adhoc_data
[2024-06-27T04:39:08.509+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-27T04:39:08.509+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-27T04:39:08.519+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_final, task_id=insert_new_record_into_audit_table, run_id=scheduled__2024-06-27T04:37:00+00:00, execution_date=20240627T043700, start_date=20240627T043908, end_date=20240627T043908
[2024-06-27T04:39:08.555+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-27T04:39:08.617+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-27T04:39:08.621+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
